deploy/helm/sumologic/templates/logs/collector/common/serviceaccount.yaml

```diff
@@ -7,6 +7,10 @@ metadata:
   labels:
     app: {{ template "sumologic.labels.app.logs.collector.serviceaccount" . }}
     {{- include "sumologic.labels.common" . | nindent 4 }}
+  {{- if .Values.sumologic.serviceAccount.annotations }}
+  annotations:
+  {{ toYaml .Values.sumologic.serviceAccount.annotations | indent 2 }}
```

Comment:
Was the values.yaml already updated with this key? Also we probably need this for other resources that have service accounts too.